src/java.base/share/conf/security/java.security

```diff
@@ -655,7 +655,8 @@ jdk.certpath.disabledAlgorithms=MD2, MD5, SHA1 jdkCA & usage TLSServer, \
 # implementations.
 
 jdk.security.legacyAlgorithms=SHA1, \
```

Comment:
Since we are now warning about weak symmetric key algorithms, we should make the description of this property more general. I would change lines 641-2 to "Legacy cryptographic algorithms and key lengths".